### ISSUE TITLE
Problem with start courseplay on contract field

### ISSUE THREAD

**Issue by SirAdams** [Courseplay/courseplay#6066]
I have the same problem. v.054 and v.056
Field 5, Contract work. 
![20200912050551_1](https://user-images.githubusercontent.com/2748305/92985748-b9b31280-f4b5-11ea-8453-8069991029ce.jpg)
[log.txt](https://github.com/Courseplay/courseplay/files/5212101/log.txt)

_Originally posted by @SirAdams in https://github.com/Courseplay/courseplay/issues/6058#issuecomment-691392711_

**Comment by pvaiko** [comment]
Fixed in 58

**Comment by SirAdams** [comment]
Today I decided to test a few mods and it turned out that the problem is when I have the AutoDrive v 1.1.0.5-RC mod loaded.
After uploading the previous version 1.1.0.4, I could not start the game and I had to delete the information about AutoDrive from the game save directory because the save after loading blocked the game in the loading image.
After deleting 2 files, the game started and there are no problems with CoursePlay at the moment.
